.gitignore

```diff
@@ -0,0 +1,3 @@
+.terraform.lock.hcl
+.terraform
+.idea
```

Comment:
Trailing newline should be added to .gitignore as well

Reply:
fixed